packages/fileeditor-extension/schema/plugin.json

```diff
@@ -9,6 +9,17 @@
         "autoClosingBrackets": {
           "type": "boolean"
         },
+        "fontFamily": {
+          "type": ["string", "null"]
+        },
+        "fontSize": {
+          "type": ["integer", "null"],
+          "minimum": 1,
+          "maximum": 100
```

Comment:
Is there any particular reason you chose this maximum number?

Reply:
I think an upper limit to `fontSize` helps catching typos early.
But no, there is no particular reason to choose `100`, except for being a round number, the same chosen by Atom editor and big enough to be confident no one wants larger fonts.